packages/near-fast-auth-signer/src/utils/multi-chain/chains/Bitcoin/Bitcoin.ts

```diff
@@ -428,12 +375,11 @@ export class Bitcoin {
       },
     };
 
-    // TODO: it should be done in parallel,
-    // but for now it's causing nonce issues on the signDelegate so we will run sequentially to avoid the issue for now
-    for (let index = 0; index < inputs.length; index += 1) {
-      // eslint-disable-next-line no-await-in-loop
-      await psbt.signInputAsync(index, mpcKeyPair);
-    }
+    await Promise.all(
```

Comment:
I think there is a rebase issue here, we can't send the requests in parallel because of nonce issues. I talked with @hcho112 and we will create a separated issue for it